components/widgets/popover/index.tsx

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {Popover as BSPopover} from 'react-bootstrap';
+
+export type Sizes = 'xs' | 'xsmall' | 'sm' | 'small' | 'medium' | 'lg' | 'large';
```

Comment:
we are using for other things the set of `'xs' | 'sm' | 'md' | 'lg' | 'xl'` why don't we try to adjust to that same aproach? I think here we are leaking part of the Bootstrap Popover.